### PR TITLE
added method moveRobot

### PR DESCRIPTION
**Pull request by MaksymMatuliak**

### CODE REVIEW

File: src/main/java/core/basesyntax/FieldXY.java [hunk 1]
```diff
@@ -13,7 +13,47 @@
...
+            }
+        } else {
+            if (robot.getDirection() == Direction.UP) {
+                robot.turnRight();
+            } else {
+                while (robot.getDirection() != Direction.RIGHT) {
+                    robot.turnLeft();
+                }
+            }
+        }
```
Comment (Tarabarnik) [inline src/main/java/core/basesyntax/FieldXY.java]: this is hard to understand what's going on here. try to use `switch/case` with robot's direction as a parameter

File: src/main/java/core/basesyntax/FieldXY.java [hunk 2]
```diff
@@ -13,7 +13,47 @@
...
+                robot.turnRight();
+            } else {
+                while (robot.getDirection() != Direction.UP) {
+                    robot.turnLeft();
+                }
+            }
+        }
+        while (robot.getY() != toY) {
+            robot.stepForward();
+        }
```
Comment (Tarabarnik) [inline src/main/java/core/basesyntax/FieldXY.java]: same

File: src/main/java/core/basesyntax/FieldXY.java (hunk 1 above)
Comment (MaksymMatuliak) [reply src/main/java/core/basesyntax/FieldXY.java]: done

File: src/main/java/core/basesyntax/FieldXY.java (hunk 2 above)
Comment (MaksymMatuliak) [reply src/main/java/core/basesyntax/FieldXY.java]: done

**Pull request closed without merging**